### ISSUE TITLE
Attempt to index local 'quality' (a nil value)

### ISSUE THREAD

**Issue by florianschanda** [Helfima/helmod#586]
I've started a fresh Pyanodon run and I am getting the following error whenever I try to add a production chain:
```
__helmod__/data/ModelCompute.lua:662: attempt to index local 'quality' (a nil value)
__helmod__/data/ModelCompute.lua:302: attempt to perform arithmetic on field 'pollution_amount' (a nil value)
```

Mods enabled:
* Elevated Rails
* Helmod (2.0.13)
* PyAL 3.0.31
* PyAE 3.1.13
* PyCoal 3.0.26
* PyFusion 3.0.8
* PyHighTech 3.0.8
* PyIndustry 3.0.9
* PyPetroleum 3.0.9
* PyPostProcessing 3.0.18
* PyRawOres 3.0.9

Steps to reproduce:
* Start new game with py recommended settings
* Open helmod
* Click "Add recipe"
* Click "Helmod button"
* Click "Raw coal" or similar

**Comment by Helfima** [comment]
you can use wrench top icon of helmod panel to report

**Comment by florianschanda** [comment]
I'll remember to use that button next time. Thanks for looking into it.

**Comment by Helfima** [comment]
yes more simple for you

**Comment by Dumpinieks** [comment]
Also encountered the bug, and also all the pyanodons mods in my game

**Comment by florianschanda** [comment]
The default setting for Py turns pollution off.

**Comment by Helfima** [comment]
fixed in v2.0.14
first error cause the pollution error, I mean

**Comment by florianschanda** [comment]
I can confirm it's fixed. Thank you for you work! :)
